src/ui/components/Library/Team/View/Tests/hooks/useTests.ts

```diff
@@ -24,3 +30,92 @@ export function useTests(): Test[] {
   );
   return value;
 }
+
+export function useTest(testId: string) {
```

Comment:
Wasn't sure how to use the existing cache for something that follows a progressively-loaded pattern, so just resorted to a simpler useQuery here for now